packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -18,15 +18,21 @@ export enum TableCellInspectorMode {
 }
 
 interface TableCellInspectorProps {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  value: string;
+  value: unknown;
```

Comment:
I went ahead with making this type `unknown` - lmk if there was a reason you hesitated to do this, I feel like it's truthful.